app/serializers/generic_event_serializer.rb

```diff
@@ -14,4 +14,14 @@ class GenericEventSerializer
   attribute :event_type do |object|
     object.type.try(:gsub, 'GenericEvent::', '')
   end
+
+  attribute :details do |record, _params|
+    record.details.deep_dup.tap do |details|
```

Comment:
I'm using `dup` because I don't want to mutate the underlying attributes that are referenced when generating the relationships in the serializer. Our serializer gem sequence goes:

1. render attributes ->
2. render relationships

In terms of the `deep_` in  `deep_dup` we never nest attributes in the event details jsonb (this was actually one of the desired changes in the v2 events model). Regardless, I'm doing a `deep_dup` rather than a `dup` to avoid breaking future event implementations that are outside of my control since I won't be here.